src/routes/oklab/+page.svelte

```diff
@@ -0,0 +1,90 @@
+<script lang="ts">
```

Comment:
Added a new `/oklab` page to test this out. Just a copy of the original RGB demo, just with the obvious colorspace change